packages/devtools/lib/src/debugger/debugger.dart

```diff
@@ -1071,6 +1155,15 @@ class BreakpointsView implements CoreElementView {
   }
 }
 
+const int DOM_VK_RETURN = 13;
```

Comment:
move these to a more central location. Expect this needs to be used anywhere we want keyboard navigation.

Reply:
Done.